scripts/gh-predeploy.js

```diff
@@ -350,7 +459,93 @@ function prepareToDeployExamples(done) {
   }))(done);
 }
 
+/**
+ * Create the index page for the blockly-samples GitHub Pages site.
+ * This page has some nice wrappers, a search bar, and a link to every plugin or example
+ * specified in in `index.md`.
+ */
+function createIndexPage() {
+  const initialContents = fs.readFileSync(`./gh-pages/index.md`).toString();
+
+  const converter = new showdown.Converter();
+  converter.setFlavor('github');
+  const text = initialContents;
+  const htmlContents = converter.makeHtml(text);
+  console.log(htmlContents);
```

Comment:
Just checking if this console.log is intentional.  It's developer-side, so no problem, but want to verify it wasn't some forgotten debugging log.